control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1879,40 +1933,68 @@ func (r *reconciler) ensureIngressControllersRemoved(ctx context.Context, hcp *h
 	}
 
 	if len(errs) > 0 {
-		return false, fmt.Errorf("failed to delete ingress resources on infra cluster: %w", errors.NewAggregate(errs))
+		return false, false, fmt.Errorf("failed to delete ingress resources on infra cluster: %w", errors.NewAggregate(errs))
 	}
 
-	return false, nil
+	return false, false, nil
 }
 
-func (r *reconciler) ensureImageRegistryStorageRemoved(ctx context.Context) (bool, error) {
+func (r *reconciler) ensureImageRegistryStorageRemoved(ctx context.Context) (bool, bool, error) {
```

Comment:
why is this function returning another bool now, can you please document it?

Reply:
sure, I will